src/components/LoggedUserCard/index.tsx

```diff
@@ -1,37 +1,37 @@
 import React from 'react'
 import styled from 'styled-components'
-import {IUser} from "../../App";
-
-interface IProps {
-    user: IUser
-    maxLikes: number
-}
+import {AppContextConsumer} from "../../Context";
 
 const UserName = styled.strong`
     color: white;
 `
 
-
-const LoggedUserCard = ({user, maxLikes}: IProps) => {
-    return (
-        <div className="media">
-            <figure className="media-left">
-                <p className="image is-32x32">
-                    <img className="is-rounded" src={user.imageUrl} />
-                </p>
-            </figure>
-            <div className="media-content">
-                <div className="content">
-                    <p>
-                        <UserName>{user.name}</UserName>
-                    </p>
+const LoggedUserCard: React.FunctionComponent = () => (
+    <AppContextConsumer>
+        {context => {
+            const {user, maxLikes} = context
+            return (
+                <div className="media">
+                    <figure className="media-left">
+                        <p className="image is-32x32">
```

Comment:
content is not a paragraph, and is it e good idea put p inside figure?

Reply:
it comes from Burma library